src/OmniSharp.Host/CommandLineApplication.cs

```diff
@@ -98,7 +98,7 @@ private void DebugAttach()
         {
             if (Debug)
             {
-                Console.WriteLine($"Attach debugger to process {Process.GetCurrentProcess().Id} to continue...");
+                // Console.WriteLine($"Attach debugger to process {Process.GetCurrentProcess().Id} to continue...");
                 while (!Debugger.IsAttached)
```

Comment:
We need to do something about this.

Reply:
Yes, it breaks LSP (over stdio) sadly :(

Reply:
i would just remove the line completely.
if you run with the feature to wait a debugger enabled
a) you should know what you are doing
b) you should be able to find the correct process yourself without the PID

Reply:
or, alternatively, check if LSP flag is enabled, and don't print it, otherwise print it